modules/core/src/main/scala/org/tessellation/domain/trust/storage/configuration.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.domain.trust.storage
+
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.NonNegLong
+
+object configuration {
+
+  val snapshotInterval: NonNegLong = 1000L
+
+  val delayToUpdateCurrent: NonNegLong = 500L
+
+}
```

Comment:
it should be either local for the algorithm or inside the application configuration (see: cli, method)

Reply:
Moved to `org.tessellation.cli.method`

Reply:
Since the domain parts of `TrustStorage` were moved to the `sdk` module, these two variables were moved into the `org.tessellation.sdk.domain.trust.storage.OrdinalTrustMap` companion object.